src/components/nav_bar/Navbar.test.tsx

```diff
@@ -12,8 +12,10 @@ import Navbar from './Navbar';
  * - Clicking auth options should open authentication drawer.
  */
 
+afterEach(() => cleanup);
+
 describe('Navbar', () => {
-    test('Should render at all times', () => {
+    test('should render at all times', () => {
         const { queryByTestId } = render(
```

Comment:
The author of `@testing-library` discourages this pattern. instead, use `screen.queryByTestId`. https://kentcdodds.com/blog/common-mistakes-with-react-testing-library.

Second, _I_ typically encourage finding things in a similar way a user would.  Would a user interact with the `Navbar` using `testid`s? Or would a user interact with it using things like text, or buttons or links?